### ISSUE TITLE
Logging DASH re-buffering events

### ISSUE THREAD

**Issue by pwalkerz** [gpac/gpac#205]
Hi there, I'm trying to log dash buffering event, meaning the event when the video stream buffer is depleted and the playback is paused due to re-buffering.
Is there any way to log those events, they usually appear in the console this way:
<Buffer : | ============= | (10/100) >
etc.
guess its showing that 10% of the current segment are buffered already. 
However I'm especially interesed in how long those buffering events are and if there are any logtools already logging this in a better way than the raw realtime console output.
For example -logs dash@debug will always show me:

[MPD_IN] Had to wait for 0 ms for the only cache file to be downloaded
[MPD_IN] Waited 0 ms - Elements in cache: 1/3
    Cache file name gmem://584026@0xa64690
    segment start time 0 sec

However 0ms seems rather short right ;-)
Also the Player option -log-utc doesn't seem to work. 
The cache logtool also doesn't seem to give me any viable information about the actual video stream buffering happening.

If I had to write my own log tool where I should start if I'd want to log re-buffering events, like logging the time the player has to pause?

ps: I'm still using version 0.5.1-DEV-rev5265M , don't know if anything has changed in newer versions

**Comment by pwalkerz** [comment]
Well things I found out:
- time logging seems to work in combination with -lf , not if only command line output is used
- they is the SyncLayer log which basically should show you buffering event in the form of "[SyncLayer] ES1: buffering on/off at xxx" however this only seems to work for the first initial buffering, all later buffering event don't seem to be shown. When I force the player into buffer depletion (i.e. running a download in parallel that uses all bandwidth) the SyncLayer buffer will get stuck at 41ms. Even after turning off the parallel download the Player seems to work normal again bust still no indication of buffering, SyncLayer still shows 41ms buffer.  
- it doesn't help to set a ReBufferLength in .gpacrc 
- part of the problem is probably that MPD_ServiceCommand(GF_NET_CHAN_BUFFER) sets com->buffer.max to the minBuffer value of the MPD. However that values should be set to com->buffer.min if I'm not mistaken (this still hasn't changed in the current version)

tl:dr : ther is no way to conveniently log re-buffering event so is it clear when rebuffering and player pauses occur as changing the ReBufferLength value in the config doesn't seem to have any effect...

**Comment by rbouqueau** [comment]
> time logging seems to work in combination with -lf , not if only command line output is used

Side note: fixed in 1ede7f5cd45c1b42c03da91f21a4e1160e13e1f9
